.travis.yml

```diff
@@ -47,13 +47,14 @@ install:
 
 script:
   - export PATH="${PATH}:$(pwd)/bin"
-  - stack build --flag dhall-json:gpl --copy-bins --local-bin-path ./bin
+  - stack build dhall-json dhall-yaml --copy-bins --local-bin-path ./bin
   - source .travis-functions.sh
   - tar -jcvf $(mk_release_name dhall) bin/dhall
-  - tar -jcvf $(mk_release_name dhall-json) bin/dhall-to-json bin/dhall-to-yaml bin/json-to-dhall bin/yaml-to-dhall
+  - tar -jcvf $(mk_release_name dhall-json) bin/dhall-to-json bin/dhall-to-yaml bin/json-to-dhall
   - tar -jcvf $(mk_release_name dhall-bash) bin/dhall-to-bash
   - tar -jcvf $(mk_release_name dhall-lsp-server) bin/dhall-lsp-server
   - tar -jcvf $(mk_release_name dhall-nix) bin/dhall-to-nix
```

Comment:
I don't understand how we're installing the binaries for `dhall-to-nix`, `dhall-lsp-server` etc. I only see the `--copy-bins` for `dhall-json` so far. Does anyone know?

Reply:
Indeed Travis fails now with:

```
$ tar -jcvf $(mk_release_name dhall) bin/dhall
tar: bin/dhall: Cannot stat: No such file or directory
tar: Error exit delayed from previous errors.
The command "tar -jcvf $(mk_release_name dhall) bin/dhall" exited with 1.
3.75s$ tar -jcvf $(mk_release_name dhall-json) bin/dhall-to-json bin/dhall-to-yaml bin/json-to-dhall
a bin/dhall-to-json
a bin/dhall-to-yaml
a bin/json-to-dhall
The command "tar -jcvf $(mk_release_name dhall-json) bin/dhall-to-json bin/dhall-to-yaml bin/json-to-dhall" exited with 0.
0.03s$ tar -jcvf $(mk_release_name dhall-bash) bin/dhall-to-bash
tar: bin/dhall-to-bash: Cannot stat: No such file or directory
tar: Error exit delayed from previous errors.
The command "tar -jcvf $(mk_release_name dhall-bash) bin/dhall-to-bash" exited with 1.
0.03s$ tar -jcvf $(mk_release_name dhall-lsp-server) bin/dhall-lsp-server
tar: bin/dhall-lsp-server: Cannot stat: No such file or directory
tar: Error exit delayed from previous errors.
The command "tar -jcvf $(mk_release_name dhall-lsp-server) bin/dhall-lsp-server" exited with 1.
0.03s$ tar -jcvf $(mk_release_name dhall-nix) bin/dhall-to-nix
tar: bin/dhall-to-nix: Cannot stat: No such file or directory
tar: Error exit delayed from previous errors.
The command "tar -jcvf $(mk_release_name dhall-nix) bin/dhall-to-nix" exited with 1.
```

Could it be that we have distributed ancient cached binaries for OS X so far?

Reply:
No, it was just https://github.com/dhall-lang/dhall-haskell/commit/96e694d7c6368a327a24dc31e8dd870ae76e7272 that restricted the binary build to `dhall-json`.